test cases/common/246 install_emptydir/meson.build

```diff
@@ -0,0 +1,4 @@
+project('install_emptydir')
+
+install_emptydir(get_option('datadir')/'new_directory', install_mode: 'rwx------')
+install_emptydir(get_option('datadir')/'new_directory/subdir', install_mode: 'rwxr-----')
```

Comment:
There should be an `install_emptydir(get_option('datadir')/'foo/bar/baz')` to test that it creates the intermediary path segments also.

Reply:
It already creates datadir (`usr/local/share/`).